react-pr2/src/features/Cart/index.jsx

```diff
@@ -0,0 +1,204 @@
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Container } from '@mui/material';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import { red } from '@mui/material/colors';
+import IconButton from '@mui/material/IconButton';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Tooltip from '@mui/material/Tooltip';
+import { makeStyles } from '@mui/styles';
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { formatPrice, thumbnailURL } from '../../utils/utils';
+import CartBreadcrumb from './components/CartBreadcrumb';
+import CartEmpty from './components/CartEmpty';
+import EnhancedTableHead from './components/EnhancedTableHead';
+import EnhancedTableToolbar from './components/EnhancedTableToolbar';
+import { cartItemCountSelector } from './selectors';
+import Quantity from './components/Quantity';
+import { removeFromCart } from '../../store/Slice/cartSlice';
+import { useTranslation } from 'react-i18next';
+import './index.scss';
+
+const useStyles = makeStyles((theme) => ({
+  root: {},
+  salePrice: {
+    marginRight: '8px',
+    fontWeight: 'bold',
+  },
+  originalPrice: {
+    marginRight: '8px',
+    fontSize: '12px',
+    textDecoration: 'line-through',
+  },
+  thumbnail: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  totalItem: {
+    color: red[400],
+  },
+}));
+
+export default function CartFeature() {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+  const [selected, setSelected] = useState([]);
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(5);
+  const { t } = useTranslation();
+
+  const cartItemsCount = useSelector(cartItemCountSelector);
+  const cartItem = useSelector((cartItem) => cartItem.cart.cartItem);
```

Comment:
```suggestion
  const cartItems = useSelector(cartItemsSelector);
```
Này số nhiều chứ nhỉ, với lại viết 1 hàm selector cho nó luôn nhé